pkg/virtctl/credentials/removekey/removekey_test.go

```diff
@@ -239,7 +242,7 @@ var _ = Describe("Credentials", func() {
 
 	It("should remove key from secret with key from file", func() {
 		filename := filepath.Join(GinkgoT().TempDir(), "test-key-file")
-		Expect(os.WriteFile(filename, []byte(testKey), 0666)).To(Succeed())
+		Expect(os.WriteFile(filename, []byte(testKey), 0600)).To(Succeed())
```

Comment:
How is this related to the linter coverage?

Reply:
done